styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -370,6 +377,13 @@ static EnvVar envVar(String name, String value) {
     env.put(TRIGGER_TYPE, runSpec.trigger().map(TriggerUtil::triggerType).orElse(null));
     env.put(ENVIRONMENT, styxEnvironment);
     env.put(LOGGING, "structured");
+    executionEnvVars.forEach((key, value) -> {
+      if (env.containsKey(key)) {
+        LOG.error("Key already exists in execution environment variables {}. Key will be skipped", key);
```

Comment:
```suggestion
        LOG.info("Key already exists in execution environment variables {}. Key will be skipped", key);
```
we should only log error if we are going to avoid or fail the execution, but here we are just ignoring and continuing